mitodl/ocw-studio
# added command to migrate testimonials to stories

#### Pre-Flight checklist

#### What are the relevant tickets?
Closes Part of issue https://github.com/mitodl/ocw-hugo-themes/issues/614

#### What's this PR do?
Adds a management command for migrating testimonials to stories

#### How should this be manually tested?
-  Create a new site with ocw-www starter
- Create few testimonials
- Update the  config  to https://github.com/mitodl/ocw-hugo-projects/blob/wassaf/rename-testimonials/ocw-www/ocw-studio.yaml for the ocw-www starter
- Verify stories appear and testimonials dont appear in the UI
- Run the following command `update_content_fields ocw-www`
- Verify stories in populated in the UI
- Run command for content sync and publish
- Verify content is published in content/stories
